lib/BloqadeExpr/Project.toml

```diff
@@ -11,7 +11,11 @@ LaTeXStrings = "b964fa9f-0449-5b57-a5c2-d3ea65f4040f"
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
 LuxurySparse = "d05aeea4-b7d4-55ac-b691-9e7fabb07ba2"
 MLStyle = "d8e11817-5142-5d16-987a-aa16d5891078"
+ParallelMergeCSR = "c4b929cc-6141-4ba4-8849-280a863346e1"
```

Comment:
not right now but we should consider moving these to an extension to reduce the package size thus the latency when we swap out BloqadeExpr with BloqadePulse